src/main/java/com/google/alpollo/AuthenticationServlet.java

```diff
@@ -1,16 +1,21 @@
 package com.google.alpollo;
 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
-import com.google.appengine.api.users.UserService;
 
 @WebServlet("/auth")
 public class AuthenticationServlet extends HttpServlet {
     @Override
     protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         UserService userService = UserServiceFactory.getUserService();
+        String loginUrl = userService.createLoginURL("/");
+        // Redirect to the login page and then to the main page
+        response.sendRedirect(loginUrl);
```

Comment:
We also won't force users to login, if they don't want to, so there's no need to redirect them to the login page.